sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasQueryParameters.cs

```diff
@@ -15,35 +17,37 @@ namespace Azure.Storage.Sas
     /// </summary>
     public sealed class BlobSasQueryParameters : SasQueryParameters
     {
+        internal UserDelegationKeyProperties _keyProperties;
+
         /// <summary>
         /// Gets the Azure Active Directory object ID in GUID format.
         /// </summary>
-        public string KeyObjectId => _keyObjectId;
+        public string KeyObjectId => _keyProperties?._objectId;
 
         /// <summary>
         /// Gets the Azure Active Directory tenant ID in GUID format
         /// </summary>
-        public string KeyTenantId => _keyTenantId;
+        public string KeyTenantId => _keyProperties?._tenantId;
 
         /// <summary>
         /// Gets the time at which the key becomes valid.
         /// </summary>
-        public DateTimeOffset KeyStartsOn => _keyStart;
+        public DateTimeOffset KeyStartsOn => (DateTimeOffset)_keyProperties?._startsOn;
```

Comment:
What does this return if not present?  What's `(DateTimeOffset)null`?

Reply:
Exception - fixed.